playwright/navigation.spec.ts

```diff
@@ -45,11 +45,6 @@ test('navigation test', async ({ page }) => {
   await expect(page.getByAltText('cat sleeping').first()).toBeVisible();
 
   await page.getByRole('link', { name: 'animals' }).click();
-  await expect(page).toHaveURL('http://localhost:3000/login?returnTo=/animals');
```

Comment:
I think this `expect(page).toHaveURL(...` shouldn't be removed completely, based on the temporary workaround commits here:

- https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/commit/f6ddf1facc92efd26b82a42c64b43cb151a0bda3
- https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/commit/38dc1858298c5ee5942f4345dea6ac7724b9b572

Reply:
34b3d22

Reply:
3cce018

Reply:
7876d34